### DISCUSSION TITLE
WhatsApp Beta Support?

### DISCUSSION THREAD

**Discussion by sudoAlphaX** [tuhinpal/WhatsBot#42] [Q&A]
WhatsApp introduced the new Web portal for the beta. Is this Userbot supported?

**Comment by tuhinpal** [comment]
Not tested new beta release 😶

**Reply by sudoAlphaX** [reply]
okay. it is not working tho. Can you please test it and release the new version :)

**Reply by tuhinpal** [reply]
Still not have the version to test it

**Comment by Hex231** [comment]
Bro, Please add Button and List like example.js of whatsapp-web.js , it will give feels of perfect Userbot. :)
